insights/client/__init__.py

```diff
@@ -344,11 +344,11 @@ def collect(self, **kwargs):
             logger.debug('Check timestamp is True.')
             cached_results = self._cached_results()
             if cached_results:
-                logger.info("Using cached collection: %s", cached_results)
+                logger.debug("Using cached collection: %s", cached_results)
```

Comment:
I think we were wanting the fact that a cached collection was being used to be info level because customers might care more about that.

Reply:
Correct. This is related to changing the way the logging output works for debugs vs infos (when in --verbose mode). This was changed this way so that --to-json and --to-stdout would work as intended without additional random logging occurring. Logging will have to be fixed. I will revert these to .infos()